src/main/java/ru/itmo/sd/deadliner2bot/bot/BotImpl.java

```diff
@@ -20,33 +26,42 @@
     @Value("${bot.token}")
     private String botToken;
 
+    private final ChatStateService chatStateService;
+
     private final InputFile defaultResponse = new InputFile(
-            "CAACAgIAAxkBAAMcYa8znOHiYPmD3vEwdk7xgmNZ7gUAAqcAAxgdBg_zfF33TbN4HyIE");
+            "CAACAgIAAxkBAAMRYbJM3wABmFg_wMNH_vYufdrHebfXAAK4AAMTF8olDES8JhyUUCMjBA");
 
     @Override
     public void onUpdateReceived(Update update) {
-        String chatId;
+        long chatId;
         String messageText;
 
         if (update.getMessage() != null) {
-            chatId = update.getMessage().getChatId().toString();
+            chatId = update.getMessage().getChatId();
             messageText = update.getMessage().getText();
         } else {
-            chatId = update.getChannelPost().getChatId().toString();
```

Comment:
As a base, we had only handlers for messages and channel posts. In fact, there may be way more corner cases (for example, sending a sticker to the bot gives us null when `getMessage()` or `getChannelPost()` are called.
Since we want to use direct messages only, I suggest only attempting to read message from `.getMessage()`